ビュー画面の実装

chamomile-jasmine opened the pull request:
# WHAT
ビュー画面を作成しました。
・ルーティングの記述
・コントローラの設定
・gemの追加（Hamlやfont-awesome-railsのgemを導入）
・Hamlの記述
・SCSSの記述


# WHY
・chat-spaceを動作させるために必要なビュー画面を作成しました。
・機能の実装はこれから行っていくとし、ビューを表示するために必要な仮置きのルーティングとコントローラーを作成しました。
設計に対して正しく機能が実装できるように意識してビューを作成しました。今回は一人での開発ですが、最初にビューを作成するとチームで開発する際に分担して機能を実装しやすいと思います。
また、font-awesome-railsを使用し、シンプルなアイコンを追加しました。


[![Image from Gyazo](https://i.gyazo.com/cdfd1d3e916081d734dbc6416450d66c.png)](https://gyazo.com/cdfd1d3e916081d734dbc6416450d66c)

chamomile-jasmine committed:
create-chat-view-by-haml-and-scss2

・ルーティングの記述
・コントローラの設定
・（Haml他）gemの追加
・Hamlの記述
・SCSSの記述
を行いました。

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットは古い記法なので修正しましょう

chamomile-jasmine replied:
シンボル型の記述方法に変更しました。

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+    :css
+      /* Email styles need to be inline */
+  %body
+    = yield
```
mailer系のファイルは使用しないので消して大丈夫です

chamomile-jasmine replied:
ありがとうございます。mailer.html.hamlを削除しました。

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
こちらも

chamomile-jasmine replied:
mailer.text.hamlも削除しました。

chamomile-jasmine committed:
Update-create-chat-view-by-haml-and-scss2

・ハッシュロケットを使用していた記述をシンボル型に修正しました（処理が遅くなるのを防ぐため）→ブラウザでの表示を確認済みです。
・今後使用しないmailer系のファイルを削除しました。

TOMARUMARU reviewed (approved):
LGTM

chamomile-jasmine commented:
ありがとうございます！
